### PR TITLE
Bugfix/#54: JS issue with extlink-window.js

### PR DESCRIPTION
**Pull request by kzenmatthias**
Fix for: https://github.com/openfed/kiso/issues/54

### CODE REVIEW

File: js/sources/extlink-window.js [hunk 1]
```diff
@@ -95,7 +95,7 @@
       }
       else {
         this.linkClass = (this.isEmail) ? 'mailto' : 'ext';
-        this.ariaLabel = (this.isSvgImage) ? this.iconElement.getAttribute('aria-label') : this.iconElement.querySelector('span.visually-hidden').textContent;
+        this.ariaLabel = this.iconElement.getAttribute('aria-label');
```
Comment (smillart) [inline js/sources/extlink-window.js:98]: @kzenmatthias, are you sure that `this.iconElement.getAttribute('aria-label')` return the expected value for both the _SVG_ and the _FontAwesome_ icon element?

Comment (kzenmatthias) [reply js/sources/extlink-window.js:98]: I've tested both scenarios and they returned both the aria-label.

Comment (smillart) [reply js/sources/extlink-window.js:98]: Could you please mention here in comment the return values for the following contexts:

1. **External links:** `[value]`,
2. **External links opening in a new window:** `[value]`,
3. **Same domain (internal) links opening in a new window:** `[value]`.

Comment (kzenmatthias) [reply js/sources/extlink-window.js]: I didn't know about the other type of links.
I pushed a fixed for the above contexts that output this values:

With an svg and fa:
- (link sends email)
- (link is external)
- (link is external and opens a new window)
- Nothing

Comment (smillart) [reply js/sources/extlink-window.js]: Looks OK for the the first three returned values.
Regarding the "**Same domain (internal) links opening in a new window**" fourth context, you have to enable the "_Include links having the same domain and opening in a new window_" checkbox option in the theme appearance _JavaScript_ settings, this option is not enabled by default.

Comment (kzenmatthias) [reply js/sources/extlink-window.js]: The internal links in the new tab gives this result:
- internal (link opens a new window)

Comment (smillart) [reply js/sources/extlink-window.js]: Looks great I think! Thanks @kzenmatthias!

**Review by smillart — APPROVED** [review]
Seems OK for merging!

**Review by rutiolma — APPROVED** [review]
Looks good to be merged.

**Pull request merged**